### PR TITLE
Removed ".playground" suffix from the Downloads

### PR DESCRIPTION
**Pull request by alexito4**
Your downloads link always have a ".playground" suffix:

Screenshot 2020-02-27 at 20 34 03

I think this made sense in the beginning when your downloads were always Playgrounds. But recently, with the composable architecture series, that's not longer the case. 

Maybe is just me, but I was a little confused when clicking on a link that says "playground" and found a GitHub page with no playground in it :P

Maybe you can consider this small change 😉 

Screenshot 2020-02-27 at 20 36 56

(as described here https://github.com/pointfreeco/pointfreeco/issues/561#issuecomment-592128341 I was having issues with the test, but I think the snapshots are updated properly now.)

### CODE REVIEW

**Comment by stephencelis** [comment]
Ha that's true! We'll chat about it

File: Sources/Views/EpisodePage.swift [hunk 1]
```diff
@@ -174,7 +174,7 @@ private func downloadsView(codeSampleDirectory: String) -> Node {
         .href(gitHubUrl(to: GitHubRoute.episodeCodeSample(directory: codeSampleDirectory))),
         .class([Class.pf.colors.link.yellow, Class.margin([.mobile: [.left: 1]]), Class.align.middle])
       ],
-      .text("\(codeSampleDirectory).playground")
+      .text("\(codeSampleDirectory)")
```
Comment (stephencelis) [inline Sources/Views/EpisodePage.swift]: I think we can even pass this directly now!
```suggestion
      .text(codeSampleDirectory)
```

**Comment by stephencelis** [comment]
Thanks for taking the time to get things running and making this PR 👍 Welcome to the PF team 😂

File: Sources/Views/EpisodePage.swift (hunk 1 above)
Comment (alexito4) [reply Sources/Views/EpisodePage.swift]: You're right 😅 sorry, it was a long day ^^

**Comment by alexito4** [comment]
Thanks, I'm glad I could help :)

**Pull request merged**